docs: fix a couple of jsdoc errors

alvaromartmart opened the pull request.

iangilman commented on src/iiiftilesource.js:
```diff
@@ -444,7 +444,7 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
      * Determine whether arbitrary tile requests can be made against a service with the given profile
      * @function
-     * @param {array} profile - IIIF profile array
+     * @param {array} options - IIIF profile array
```
Looks like this isn't quite right either. If you look at the code, it's more like this:

```
     * @param {Object} options
     * @param {Array|String} options.profile
     * @param {Number} options.version
     * @param {Number} options.extraFeatures
```

I don't know enough about IIIF to know what all these parameters mean. If you do, please add some additional commentary if appropriate.

Also, I don't think this is true:

```
 * @throws {Error}
```

… At least I don't see anywhere where it would be doing that.

It does return a Boolean, though, so it might be worth adding:

```
* @return {Boolean}
```

iangilman commented:
@alvaromartmart I see you've added some commits... are you cleaning this up now?

alvaromartmart commented:
> @alvaromartmart I see you've added some commits... are you cleaning this up now?

Sorry for that, I had forgotten about this PR and updated my master branch, I've fixed that so only the original commit is present.

As for your review, to be honest I'm not familiar with IIIF tilesources, so I don't really have much to contribute for its documentation. Maybe I should leave that line untouched and just keep the other fix regarding the drawer.js.

iangilman commented:
Well, I think even just including the names of the params (per my suggestion above) would be more helpful than what we currently have, even if we can't explain what they do, so just making that change (and the others I suggest) would be an improvement. I would say that would be enough to get this merged, and then someone who's more familiar with IIIF could augment in the future. 

Sound fair?

Mayank1728 commented:
Hey @alvaromartmart ! I would love to contribute if possible.

iangilman commented:
@Mayank1728 That would be great! It looks like @alvaromartmart may have abandoned this. Would you be up for forking his changes and finishing it up?

iangilman commented:
For that matter, there are so few changes you might as well just start a fresh fork rather than forking his!

Mayank1728 commented:
> For that matter, there are so few changes you might as well just start a fresh fork rather than forking his!

Hey @iangilman ! This is the another issue I could contribute on. Please guide me on how to get started.

iangilman commented:
@Mayank1728 Wonderful! You'll want to get set up so you can build the project: 

https://github.com/openseadragon/openseadragon/blob/master/CONTRIBUTING.md

... And here's info on making a pull request: 

https://github.com/openseadragon/openseadragon/wiki/Making-a-Pull-Request

This patch is just documentation changes, so I guess technically you don't need to be able to build OSD, but it would probably be good to do so if you want to contribute additional patches. 

I recommend making a fresh fork of the current master and then making the same changes @alvaromartmart had made in this PR... It's just a few, so it's probably easier to re-do them rather than forking from his version and fixing the merge conflicts. 

From there, you just need to make the changes I suggested above. Let me know if any of them don't make sense! 

Thank you again!

Mayank1728 commented:
Hello @iangilman ! I have made the documentation changes as per the requirements however I somewhat messed things up. I have created 2 pull request for the changes. Please let me know about things I need could work on.
Thank you!

iangilman commented:
@Mayank1728 I'm not seeing the pull requests. Can you share a link to them?

iangilman commented:
This has been resolved by #2211. @alvaromartmart Thank you for getting the ball rolling!
